portfolio/src/main/java/com/google/sps/servlets/DeleteComments.java

```diff
@@ -33,21 +33,21 @@
 /** Servlet that returns some example content. */
 @WebServlet("/delete-data")
 public class DeleteComments extends HttpServlet {
-  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-  private static final String COMMENT_TABLE_NAME = "Comment";
-  private static final String TIMESTAMP_COLUMN_NAME = "submit_time";
+  private final DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+  private DataServlet dataServlet = new DataServlet();
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int numDeleteComments = 1;
-    Query query = new Query(COMMENT_TABLE_NAME ).addSort(TIMESTAMP_COLUMN_NAME, SortDirection.ASCENDING);
-    PreparedQuery results = datastore.prepare(query);
+    Query query = new Query(dataServlet.COMMENT_TABLE_NAME )
```

Comment:
Since the constants are static members of `DataServlet` - you don't need to create an instance to get at them.
```suggestion
    Query query = new Query(DataServlet.COMMENT_TABLE_NAME )
```

Can you also remove the `DataServlet` instantiation on line 37?

Reply:
oh whoops! thanks!